docs/user/reference/hook.md

```diff
@@ -283,14 +285,26 @@ information or advice before signalling the error.
 
 This section describes the workflows and associated hook events which are important to the operation of Juju.
 
-(action-hooks)=
-### Action hooks
+(actions)=
+### Actions
 
```

Comment:
Glad we're straightening things here (it always confused me why in Ops action events do not inherit from HookEvent, but this rhymes with that). But: This section is still under "Hooks in detail", so if we want to make this change we should probably move this whole section out to https://canonical-juju.readthedocs-hosted.com/en/3.6/user/reference/action/